perfkitbenchmarker/linux_benchmarks/netperf_aggregate_benchmark.py

```diff
@@ -160,24 +199,42 @@ def RunNetperfAggregate(vm, server_ips):
       login_shell=False,
       timeout=1200)
 
-  # print out netperf_tps.log to log
-  stdout_1, stderr_1 = vm.RemoteCommand(
-      f'cat {netperf.NETPERF_EXAMPLE_DIR}/netperf_tps.log',
-      ignore_failure=True,
-      should_log=True,
-      login_shell=False,
-      timeout=1200)
-
-  logging.info(stdout_1)
-  logging.info(stderr_1)
-
+  samples = []
   # do post processing step
-  proc_stdout, _ = vm.RemoteCommand(
-      f'cd {netperf.NETPERF_EXAMPLE_DIR} && ./post_proc.py '
+  if 'STREAM' in FLAGS.netperf_aggregate_benchmarks:
+    proc_stdout, _ = vm.RemoteCommand(
+      f'cd {netperf.NETPERF_EXAMPLE_DIR} && python3 post_proc.py '
+      '--intervals netperf_outbound.log',
+      ignore_failure=True)
```

Comment:
should this not ignore failure and fail benchmark if command failed?

Reply:
changed to ignore_failure=False